spec/tests/ColumnLineageDatasetFacet/1.json

```diff
@@ -1,30 +1,171 @@
+              {
+                "type": "INDIRECT",
+                "subtype": "JOIN",
+                "description": "ON (DISCOUNTS.CUSTOMERS_ID=CUSTOMERS.ID)"
+              } 
+            ]
           }
         ],
-        "transformationDescription": "",
-        "transformationType": ""
+        "transformationType": "IDENTITY"
       },
-      "order_placed_on": {
+      "STARTS_AT": {
         "inputFields": [
           {
-            "field": "order_placed_on",
-            "name": "public.top_delivery_times",
-            "namespace": "postgres://postgres:5432"
+            "namespace": "SnowflakeOpenLineage",
+            "name": "DISCOUNTS",
+            "field": "STARTS_AT",
+            "transformations": [
+              {
+                "type": "DIRECT",
+                "subType": "IDENTITY",
+                "masking": false
+              }
+            ]
+          },
+          {   
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "ID",
+            "transformations": [
+              { 
+                "type": "INDIRECT",
+                "subtype": "JOIN",
+                "description": "ON (DISCOUNTS.CUSTOMERS_ID=CUSTOMERS.ID)"
+              }
+            ]
+          },
+          {
+            "namespace": "SnowflakeOpenLineage",
+            "name": "DISCOUNTS",
+            "field": "CUSTOMERS_ID",
+            "transformations": [
+              {
+                "type": "INDIRECT",
+                "subtype": "JOIN",
+                "description": "ON (DISCOUNTS.CUSTOMERS_ID=CUSTOMERS.ID)"
+              } 
+            ]
+          }
+        ],
+        "transformationType": "IDENTITY"
+      },
+      "ENDS_AT": {
+        "inputFields": [
+          {
+            "namespace": "SnowflakeOpenLineage",
+            "name": "DISCOUNTS",
+            "field": "ENDS_AT",
+            "transformations": [
+              {
+                "type": "DIRECT",
+                "subType": "IDENTITY",
+                "masking": false
+              }
+            ]
+          },
+          {   
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "ID",
+            "transformations": [
+              { 
+                "type": "INDIRECT",
+                "subtype": "JOIN",
+                "description": "ON (DISCOUNTS.CUSTOMERS_ID=CUSTOMERS.ID)"
+              }
+            ]
+          },
+          {
+            "namespace": "SnowflakeOpenLineage",
+            "name": "DISCOUNTS",
+            "field": "CUSTOMERS_ID",
+            "transformations": [
+              {
+                "type": "INDIRECT",
+                "subtype": "JOIN",
+                "description": "ON (DISCOUNTS.CUSTOMERS_ID=CUSTOMERS.ID)"
+              } 
+            ]
           }
         ],
-        "transformationDescription": "",
-        "transformationType": ""
+        "transformationType": "IDENTITY"
```

Comment:
This is in the spec still, but I'm not sure if it's still relevant? Since whether it's an IDENTITY or not depends on how each inputField transforms?

Reply:
it's not relevant anymore, in the spec it's marked as deprecated

Reply:
updated